### DISCUSSION TITLE
Am I able to use this to build my own taskbar?

### DISCUSSION THREAD

**Discussion by Joe23232** [emilk/egui#280] [General]
Hi I am thinking of making a taskbar similar to [polybar](https://polybar.github.io/).

So I wanted to ask you using this extension, is this possible? Is it feasible as well?

**Comment by emilk** [comment]
Yeah, that should be possible. Try it :) One thing that might be missing is a fixed window position on screen, not sure how that would be handled, but it can probably be added to `egui_glium` (the default native backend for `egui`).

**Reply by Joe23232** [reply]
One other question, If I used immediate continuous mode, does it have more overhead compared to retained?

**Reply by emilk** [reply]
egui has a reactive mode where it only repaints when interacted with, so if you use that you should be fine

**Comment by Joe23232** [comment]
Interesting thanks mate :)

**Comment by Joe23232** [comment]
I see thanks mate :)
